app/test/test_ftp.py

```diff
@@ -153,23 +133,6 @@ def test_local_server_delete(self):
 
         server.terminate()
 
-    @unittest.skipUnless(os.getenv("CF_INSTANCE_GUID"), "CF-only test")
-    def test_cf_server_get(self):
-        server = multiprocessing.Process(
-            target=serve,
-            args=(self.root,),
-            kwargs=self.params
-        )
-        server.start()
-        time.sleep(5)
-        worker = FTPWorker(**self.params)
-        with worker as active:
-            items = set(i.file for i in active.get(active.filenames))
-            self.assertEqual(len(self.files), len(items))
-            self.assertEqual(set(self.files.values()), items)
-
-        server.terminate()
-
     @unittest.skipIf(os.getenv("CF_INSTANCE_GUID"), "local-only test")
```

Comment:
Same with this line, can be deleted as we'll never be in CF

Reply:
done